### ISSUE TITLE
Reinforcing accessibility of change logs

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/specberus#1928]
I've gotten comments that certain TR documents indicate changes using color alone. In most cases they are also providing a Diff or a change log (which is accessible) and that resolves the issue. But if for whatever reason there ends up being a document indicating changes with color alone, that'd be an accessibility problem. I think it would be wise for us to add an explicit mention to accessibility when it comes to providing a list of changes. For example, all places where we say:

> It must include a link to changes since the previous draft (e.g., a list of changes or a diff document or both; see the online HTML diff tool). The group must indicate which changes may affect conformance.
   
For example, adding "accessible" in front of "list" so that it says "accessible list of changes", just to add that slight remark.

**Comment by daniel-montalvo** [comment]
@deniak @plehegar @iadawn 
I just opened this as a follow-up from a sysreq.

See above for rationale.

I think this only affects docs. Overall agree with @deniak that "The Working Groups have different needs so it would be difficult to prevent a specific use case".

**Comment by plehegar** [comment]
So, we already have:
[[
The document(s) must conform to the [Web Content Accessibility Guidelines 2.1](https://www.w3.org/TR/WCAG21/), Level AA. Note: You may wish to consult the [customizable quick reference to Web Content Accessibility Guidelines 2.1](https://www.w3.org/WAI/WCAG21/quickref/).
]]
https://www.w3.org/pubrules/doc/rules/?profile=WD#wcag

Regarding diff/change log, we only have:
[[
It should include a link to changes since the previous draft (e.g., a list of changes or a diff document or both; see the [online HTML diff tool](https://www.w3.org/2007/10/htmldiff)).
]]
https://www.w3.org/pubrules/doc/rules/?profile=WD#changesList

If we start listing some specific for the use cases of diff, I'm afraid we'll end up having to list everything in pubrules. So, I'm not sure we should enhance pubrules further (the longer it is, the less likely it will be read).

If we have drafts that are not following WCAG, those should be called out. If we have many of those, then a reminder to chairs and editors might be in order. If there is a way to automatically enforce, that could be added to the pubrules checker.

**Comment by daniel-montalvo** [comment]
@plehegar sounds reasonable to me.

Will flag individual scenarios if we come across them
